esrally/rally.py

```diff
@@ -51,6 +51,8 @@
 from esrally.tracker import tracker
 from esrally.utils import console, convert, io, net, opts, process, versions
 
+OFFLINE_SUBCOMMANDS = ("compare", "list", "start", "stop", "generate")
```

Comment:
So this is complicated:

 * `start` can pull Docker images,
 * `stop` writes to the metrics store which can be a remote cluster.
 * `list races` and `list tracks` support fetching the data from a remote metrics cluster
 * `list tracks`, `list cars` and `list elasticsearch-plugins` need to fetch data from GitHub

So the commands that are truly offline are: compare, list telemetry, list pipelines, and generate.

Reply:
👍  Thanks, that's good to know. I'll make changes.

Reply:
Agree. I struggled with this one. I'll think about it some more.